components/brave_wallet/browser/zcash/rust/orchard_shard_tree_unittest.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include <memory>
+
+#include "base/files/scoped_temp_dir.h"
+#include "brave/components/brave_wallet/browser/zcash/rust/orchard_testing_shard_tree_impl.h"
+#include "brave/components/brave_wallet/browser/zcash/zcash_test_utils.h"
+
+static_assert(BUILDFLAG(ENABLE_ORCHARD));
+
+namespace brave_wallet {
+
+class OrchardShardTreeUnitTest : public testing::Test {
```

Comment:
there are two files named orchard_shared_tree_unittest.cc, I think this is orchard_shard_tree_impl_unittest?

Reply:
or I guess maybe this one is ok and the other one should be orchard_shard_tree_delegate_unittest?

Reply:
actually I have no idea what is going on with the other shard_tree_unittest

Reply:
Right one is brave/components/brave_wallet/browser/zcash/orchard_shard_tree_unittest.cc